gerber-diagrams/lib/Gerber/Diagrams.hs

```diff
@@ -20,7 +19,13 @@ import qualified Linear
 gerberToDiagram
   :: Diagrams.Renderable ( Diagrams.Path Diagrams.V2 Double ) b
   => Gerber.Evaluator ( Diagrams.QDiagram b Diagrams.V2 Double Diagrams.Any )
-gerberToDiagram =
+gerberToDiagram = gerberToDiagramCustom defaultWithPolarity
+
+
+gerberToDiagramCustom
+  :: (Typeable n, RealFloat n, Diagrams.Renderable (Diagrams.Path Diagrams.V2 n) b, Diagrams.Transformable m, Fractional (Diagrams.N m), Diagrams.V m ~ Diagrams.V2)
+  => (Polarity.Polarity -> Diagrams.QDiagram b Diagrams.V2 n Diagrams.Any -> m) -> Gerber.Evaluator m
```

Comment:
I think I would rather supply a `Config` to this rather than a full blown function - being able to do _anything_ with the diagram looks too powerful. How about:

```suggestion
data Config = Config
  { foreground :: Diagrams.Colour
  , background :: Diagrams.Colour
  }
```

gerberToDiagramCustom
  :: Diagrams.Renderable ( Diagrams.Path Diagrams.V2 Double ) b
  => Config -> Gerber.Evaluator ( Diagrams.QDiagram b Diagrams.V2 Double Diagrams.Any )
```

Reply:
@ocharles  :+1: could you put up a new PR with the modified behaviour or update this one.

Reply:
What if you would like to change the line width used too, or some other rendering property ? I am thinking of a board outline layer where I think one might want to render with a thicker stroke width.